packages/wallets/src/components/Base/WalletAlertMessage/WalletAlertMessage.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ReactNode } from 'react';
+import useDevice from '../../../hooks/useDevice';
+import ErrorCircleCrossmark from '../../../public/images/error-circle-crossmark.svg';
+import InfoCircleDots from '../../../public/images/info-circle-dots.svg';
+import SuccessCircleCheckmark from '../../../public/images/success-circle-checkmark.svg';
+import { WalletButton, WalletText } from '../index';
+import styles from './WalletAlertMessage.module.css';
+
+const typeMapper = {
+    error: {
+        color: 'error',
+        icon: ErrorCircleCrossmark,
+    },
+    info: {
+        color: 'blue',
+        icon: InfoCircleDots,
+    },
+    success: {
+        color: 'success',
+        icon: SuccessCircleCheckmark,
+    },
+};
+
+type TAlertMessageProps =
+    | {
+          buttonLabel: ReactNode;
+          message: ReactNode;
+          onClickHandler: VoidFunction;
+          type: 'error' | 'info' | 'success';
+          variant: 'with-action-button';
+      }
+    | {
+          message: ReactNode;
+          type: 'error' | 'info' | 'success';
+          variant: 'base';
+      };
+
+const WalletAlertMessage: React.FC<TAlertMessageProps> = props => {
```

Comment:
Lets not do this @heorhi-deriv. Lets list out all the props we passing. One is for better readability, second may accidentally pass in props that are not expected by the component. This can lead to unexpected behavior and buga.